src/systemtest/java/util/PropertiesHandler.java

```diff
@@ -0,0 +1,28 @@
+package util;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.InputStream;
+import java.util.Map.Entry;
+import java.util.Properties;
+
+public class PropertiesHandler {
+
+    public static void setProperties() throws Throwable {
+        final String systemTestFilePath;
+
+        if (System.getProperty("propertiesFile") != null) {
+            systemTestFilePath = System.getProperty("propertiesFile");
+            InputStream fis = new FileInputStream(new File(systemTestFilePath));
```

Comment:
I dont think you should make the variable names short. Write out what it is, it will be easier to read and understand.
I would rather call it what it is for example systemTestFileInputStream.

Reply:
done